flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,41 @@
+flink-connector-hive
+Copyright 2014-2019 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.hadoop:hadoop-common:2.7.2
+- commons-io:commons-io:2.4
+- commons-logging:commons-logging:1.1.3
```

Comment:
please sort the dependencies alphabetically

Reply:
ok